spec/API_specification/creation_functions.md

```diff
@@ -49,6 +49,42 @@ This function cannot guarantee that the interval does not include the `stop` val
 
     -   a one-dimensional array containing evenly spaced values. The length of the output array must be `ceil((stop-start)/step)`.
 
+
+(function-asarray)=
+### asarray(obj, /, *, dtype=None, device=None, copy=False)
+
+Convert the input to an array.
+
+#### Parameters
+
+-   **obj**: _Union\[ float, NestedSequence\[ bool | int | float ], SupportsDLPack, SupportsBufferProtocol, memoryview ]_
```

Comment:
Doesn't `memoryview` already support the buffer protocol? I guess it doesn't hurt to add it explicitly.

Reply:
I wasn't sure it did. It would make sense, but the first paragraphs of both https://docs.python.org/3/c-api/buffer.html and https://docs.python.org/3/library/stdtypes.html#memoryview only say `bytes` and `bytearray` do. `memoryview` wraps a buffer, but I couldn't figure out if itself exposes the buffer protocol or not (and I'm not bothered enough to go read the source code for it).

Reply:
A search for `static PyBufferProcs` turns up 7 implementations of the buffer protocol in CPython:
```
arraymodule.c
mmapmodule.c
_ctypes.c
bytesio.c
bytarrayobject.c
bytesoject.c
memoryobject.c
```

So it looks like that definitely does include memoryview.

Reply:
Thanks. I'll remove it then.